reference/configuration/framework.rst

```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/translations``
+
+This option allows to define a path where the component will write
```

Comment:
I'm not sure about this description. Should we change it for this other description?

```rst
This option allows to define the path where the application translations files
are stored.
```

Reply:
Actually it's two default things: it is where the command generates translation files and where domains take precedence on other registrations.